graceful-fs.js

```diff
@@ -366,12 +352,24 @@ function patch (fs) {
 function enqueue (elem) {
   debug('ENQUEUE', elem[0].name, elem[1])
   fs[gracefulQueue].push(elem)
+  retry()
 }
 
 function retry () {
+  if (fs[gracefulQueue].length === 0)
+    return
+
   var elem = fs[gracefulQueue].shift()
   if (elem) {
```

Comment:
if you stop when the length is 0, is there ever a case where `elem` is falsey?

Reply:
good point! i'll remove the truthy check on `elem`